### ISSUE TITLE
Inconsistent _Attack and _Release methods in PolySynth and MonoSynth 

### ISSUE THREAD

**Issue by JunShern** [processing/p5.js-sound#300]
Just realized that PolySynth and MonoSynth have inconsistent naming for the same API:
- MonoSynth uses [triggerAttack](https://github.com/processing/p5.js-sound/blob/master/src/monosynth.js#L161) and [triggerRelease](https://github.com/processing/p5.js-sound/blob/master/src/monosynth.js#L190)
- PolySynth uses [noteAttack](https://github.com/processing/p5.js-sound/blob/master/src/polysynth.js#L247) and [noteRelease](https://github.com/processing/p5.js-sound/blob/master/src/polysynth.js#L354)

I would recommend that we rename the MonoSynth methods to noteAttack and noteRelease, since these feel more intuitive (at least to me).

Happy to make the change if others agree. @therewasaguy @jvntf

**Comment by therewasaguy** [comment]
Yes it would be nice to have consistent naming here so that mono/poly synths are interchangeable.

I like noteAttack, but we have been using `triggerAttack/Release` in p5.Envelope - let's stick with that?
